daml-lf/integration-test-lib/src/main/com/daml/lf/CantonFixture.scala

```diff
@@ -71,27 +72,36 @@ trait CantonFixture
   protected lazy val nParticipants: Int = 1
   protected lazy val timeProviderType: TimeProviderType = TimeProviderType.WallClock
   protected lazy val tlsEnable: Boolean = false
+  protected lazy val enableDisclosedContracts: Boolean = false
 
   // This flag setup some behavior to ease debugging tests.
   //  If `true`
   //   - temporary file are not deleted (this requires "--test_tmpdir=/tmp/" or similar for bazel builds)
   //   - some debug info are logged.
   protected val cantonFixtureDebugMode = false
 
-  final protected val logger = org.slf4j.LoggerFactory.getLogger(getClass)
+  final protected val cantonLogger = org.slf4j.LoggerFactory.getLogger(getClass)
```

Comment:
Why the rename here? Is it to avoid a `logger` val double override or is it for clarification?

Reply:
Its a collision, StrictLogger has a `logger` field as well, and since thats a generic logger, I figured that should keep the name, and the canton one should have a more specific name

Reply:
Alternatively I could try to drop StrictLogger from whatever is using it, I don't recall off the top of my head

Reply:
Dropping `StrictLogger` (or at least getting some understanding as to why we need it) might be a good idea?

Reply:
It is fine with me to rename that.

Reply:
Turns out we could safely drop StrictLogger, so this can remain as is

Reply:
I suspect that your initial view that StringLogging is not actually needed (as another logger is in scope) is correct. I've found a similar issue elsewhere.

Reply:
Yeah, I'm now worried that the canton and test specific logs are being mixed together. It'll be hard to see whats coming from your test logic, and whats coming from the canton jar

Reply:
Good point.

The canton jar (if it's doing logging correctly), should generally log against the class that issues the log command (i.e. info, etc.). So for that logging, your concern should be managed.

(TBH, my knowledge here now gets a little hazy): any root level logging likely(?) uses the base logger we specify here, and so at this point your concern is valid. Canton should don't generally be doing any root/base level logging, so I'd suggest that this a not as high a concern as it initially feels.

Reply:
Looking further into it, we don't actually pass that logger onto the canton jar, its just used for 4 progress prints about cantons initialisation, so should be fine.